Commencement.Mvc/Views/_ViewStart.cshtml

```diff
@@ -1,3 +1,3 @@
 ﻿@{
-    Layout = "~/Views/Shared/_Layout.cshtml";
+    Layout = "~/Views/Shared/_StudentLayout.cshtml";

```

Comment:
@jSylvestre you sure you want to do this?  I would only do this on the two student pages, not the entire mvc views